src/features/amUI/common/DelegationModal/AccessPackages/AccessPackageInfo.tsx

```diff
@@ -169,6 +171,11 @@ export const AccessPackageInfo = ({ accessPackage, availableActions = [] }: Pack
                 size='xs'
                 status={actionError.httpStatus}
                 time={actionError.timestamp}
+                message={
```

Comment:
I am wondering if maybe we shouldn't display the status and timestamp in the cases where there isn't a technical error, since the http status won't actually be of use for the user and may end up distracting from the actual message because it looks more like a technical error (even if it is not). 🤔
Removing it also makes the error smaller, which is probably a plus.
Maybe either add a bool to the technical error component to hide the technical details, or not use the TechnicalParagraph component for non-technical errors, and just display the validation error sentence alone inside the alert.
![image](https://github.com/user-attachments/assets/86dcf0de-fb5d-47e0-bb9c-7872f83e39bb)